images/build/Dockerfiles/gitHubRunners.BuildPackDepsStretch.Dockerfile

```diff
@@ -1,4 +1,4 @@
 # The digest here must match with what is present in the following location
 # so that downloading of the image is very fast in GitHub workflows
 # https://github.com/actions/virtual-environments/blob/master/images/linux/Ubuntu1804-README.md
-FROM buildpack-deps:stretch@sha256:dc901bbf4b34e4ca8771c0d0773e557221452f97bcf0c732de7ecda3782bdf97
+FROM buildpack-deps:stretch@sha256:dee4275fc056551e1f83c5a3ea024510ca63f03ceedd9a1c29cbab70644b046b
```

Comment:
Noticed we are out of date with the GitHub runners's base layer

Reply:
curious: what will we do when we release buster version of this image? do we need to keep 2 different sha for buster and stretch?